<launch>
  	<include file="$(find place_recog_eval)/launch/base.launch" />

	<param name="method" value="context" type="string"/>
	<param name="leaf_size" value="0.2" type="double"/>
	<param name="threshold" value="0.038" type="double"/>

	<node pkg="place_recog_eval" type="evaluate" name="evaluate" output="screen" cwd="node"/>
</launch>
